### DISCUSSION TITLE
Any tips for full-finetuning SD3.5 Medium?

### DISCUSSION THREAD

**Discussion by GobbleCrow** [bghira/SimpleTuner#1333] [Q&A]
I recently tried to train SD3.5 Medium on a dataset that worked for SDXL (I didn't train SDXL with SimpleTuner) but haven't gotten as great results. The adherence and accuracy while trained on the same dataset are fairly lacking, I'm not sure if it's my validation settings or the model just doesn't get it. I tried to keep my settings similar to the previous training hoping that maybe that would help it out however it didn't. Keep in mind I'm also not very knowledgeable when it comes to fine-tuning and SimpleTuner so maybe it's just me doing something wrong. I have my config.json and multidatabackend.json ready if needed though just some simple guidance and tips would be much appreciated if someone has gotten something good from training this model. I've probably started over 60 trainings and still none are great. Thank you in advance!

Edit: I'm currently testing all combinations of Samplers x Schedulars x cfg x SD3Guidance in comfyui right now to see if it's just a perfect setting that'll do the trick lol. I never really could find an optimal combination for the base model let alone my fine-tuned one so it could potentially just be an issue with my validations.

**Comment by bghira** [comment]
it is sd35m's fault tbh it is too few parameters to be easily finetuned. needs a cluster of gpus

**Reply by GobbleCrow** [reply]
Really? Is that just a known thing? I thought it would be the perfect model for my case due to the image inference speed but apparently not lol. Is there any new, quick models you’d suggest? I’m using an RTX A6000.

**Reply by bghira** [reply]
the cluster of GPUs is just for training time, i've tried maybe 100 different finetunes of SD 3.5 Medium and it's the least finetunable model i've ever encountered other than Cascade or Sana.

**Reply by bghira** [reply]
probably PixArt or SDXL, though you're stuck with the 4ch VAE.

**Reply by GobbleCrow** [reply]
Ok, I’ll look into PixArt. What would you say about SD3.5 Large Turbo? Also keep in mind my entire dataset is pretty much pixel art characters in a UV format.

**Reply by bghira** [reply]
pixel art has its own challenges to train and i'm not sure a 16ch VAE even fixes them

you will definitely struggle with the 4ch VAE though

in general having clean pixel groups that don't change randomly in size is not. really doable. with latent diffusion models (LDMs)

**Reply by bghira** [reply]
and Turbo / Lightning models are not tunable, unfortunately.

**Reply by GobbleCrow** [reply]
Ok, I usually do just downscale the image by 16x “nearest” to make it pixel perfect. At the moment I’m trying to make Minecraft skins so all I really want is just a real fast model that I can get looking and producing how I want. I’ll try pixart, is it supported by simpletuner? lol.

**Reply by bghira** [reply]
yes: https://github.com/bghira/SimpleTuner/blob/main/documentation/quickstart/SIGMA.md
